src/machines/mod.rs

```diff
@@ -100,6 +103,15 @@ impl TryFrom<WorkflowCommand> for WFCommand {
                 Ok(WFCommand::CancelWorkflow(s))
             }
             workflow_command::Variant::SetChangeMarker(s) => Ok(WFCommand::SetChangeMarker(s)),
+            workflow_command::Variant::StartChildWorkflowExecution(s) => {
+                Ok(WFCommand::AddChildWorkflow(s))
+            }
+            workflow_command::Variant::RequestCancelExternalWorkflowExecution(s) => {
+                Ok(WFCommand::RequestCancelChildWorkflow(s))
+            }
+            workflow_command::Variant::SignalExternalWorkflowExecution(_) => {
+                Err(EmptyWorkflowCommandErr) // TODO
```

Comment:
This can just be a `todo!` macro to be a little more obvious until it's done

Reply:
sgtm